### PR TITLE
Apply inertia also when collisions are enabled

### PR DESCRIPTION
**Pull request by RaananW**
Fixes https://forum.babylonjs.com/t/bug-free-camera-never-stop/45546

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14504/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14504/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14504/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14504/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14504/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14504/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14504/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14504/merge#BCU1XR#0

File: packages/dev/core/src/Cameras/targetCamera.ts [hunk 1]
```diff
@@ -325,13 +325,7 @@ export class TargetCamera extends Camera {
...
         // If we have an inertia of zero, copy the values to apply and then zero out the direction vector
         // Else, scale the previous values and apply them
-        if (this.inertia === 0) {
-            TmpVectors.Vector3[0].copyFrom(this.cameraDirection);
-            this.cameraDirection.scaleInPlace(0);
-        } else {
-            this.cameraDirection.scaleInPlace(relativeInertia);
-            TmpVectors.Vector3[0].copyFrom(this.cameraDirection);
-        }
+        TmpVectors.Vector3[0].copyFrom(this.cameraDirection);
```
Comment (Popov72) [inline packages/dev/core/src/Cameras/targetCamera.ts:328]: The comment should be moved too.

File: packages/dev/core/src/Cameras/targetCamera.ts [hunk 2]
```diff
@@ -325,13 +325,7 @@ export class TargetCamera extends Camera {
 
         // If we have an inertia of zero, copy the values to apply and then zero out the direction vector
         // Else, scale the previous values and apply them
```
Comment (RaananW) [inline packages/dev/core/src/Cameras/targetCamera.ts:327]:
```suggestion
        //  copy the values to a temp vector
```

File: packages/dev/core/src/Cameras/targetCamera.ts [hunk 3]
```diff
@@ -383,6 +377,12 @@ export class TargetCamera extends Camera {
             if (Math.abs(this.cameraDirection.z) < endMovementThreshold) {
                 this.cameraDirection.z = 0;
             }
+            const relativeInertia = this._getInertiaRelativeToTime();
+            if (this.inertia === 0) {
```
Comment (RaananW) [inline packages/dev/core/src/Cameras/targetCamera.ts:381]:
```suggestion
            // If we have an inertia of zero, zero out the direction vector
            // Else, scale the previous values and apply them
            if (this.inertia === 0) {
```

File: packages/dev/core/src/Cameras/targetCamera.ts (hunk 1 above)
Comment (RaananW) [reply packages/dev/core/src/Cameras/targetCamera.ts:327]: Done :-)

**Comment by RaananW** [comment]
@PolygonalSun - the test had to be adjusted, so it might not be the best solution? or maybe the test scenario is correct (takes 2 frames more in 120fps)?

**Comment by PolygonalSun** [comment]
Closing to address in another PR

**Pull request closed without merging**